kobo/client/commands/cmd_resubmit_tasks.py

```diff
@@ -33,7 +36,7 @@ def run(self, *args, **kwargs):
         failed = False
         for task_id in tasks:
             try:
-                resubmitted_id = self.hub.client.resubmit_task(task_id, kwargs.pop("force", False))
+                resubmitted_id = self.hub.client.resubmit_task(task_id, force, priority)
```

Comment:
I think this is going to cause a compatibility issue.  This will unconditionally send 3 arguments to resubmit_task, but resubmit_task for all currently deployed services is only going to accept 2 arguments.  So, this will crash if the client upgraded their kobo version and the server is still running an older version, whether or not the client tried to use `--priority` option.  It's an important scenario since kobo is a library used by multiple clients and services which might have different requirements on the specific kobo version they want to use.

How about only sending the "priority" argument here if it's non-None?

Reply:
It makes sense. Updated.